src/java.base/unix/native/jspawnhelper/jspawnhelper.c

```diff
@@ -22,6 +22,11 @@
  * or visit www.oracle.com if you need additional information or have any
  * questions.
  */
+/*
+ * ===========================================================================
+ * (c) Copyright IBM Corp. 2019, 2019 All Rights Reserved
```

Comment:
Should this have an IBM copyright if it was copied from the OpenJDK code?

Reply:
not entirely sure... it's the way we've done it before!

Reply:
the copyright checker will complain... as it's a change...

Reply:
Please check it. It doesn't seem correct to me to add IBM copyright on code that IBM didn't author.

Reply:
hi, i'll remove, as you say it's then effectively identical to openjdk